src/main/resources/templates/javascript/exercise/src/bubblesort.js

```diff
@@ -0,0 +1,3 @@
+export default class BubbleSort {
+    // TODO: implement in performSort(Date[])
```

Comment:
Same here: In the Rust template (and the Java template), we have method bodies in the template for `performSort` / `perform_sort`. Is there any particular reason you don't add a `performSort(input)` here?